pkg/build/strategies/sti/sti.go

```diff
@@ -318,11 +314,12 @@ func (b *STI) Execute(command api.Script, request *api.Request) error {
 	defer errReader.Close()
 	defer errWriter.Close()
 	opts := docker.RunContainerOptions{
-		Image:           request.BaseImage,
-		Stdout:          outWriter,
-		Stderr:          errWriter,
-		PullImage:       request.ForcePull,
-		ExternalScripts: request.ExternalRequiredScripts,
+		Image:     request.BaseImage,
+		Stdout:    outWriter,
+		Stderr:    errWriter,
+		PullImage: request.ForcePull,
+		// TODO fix this value
+		ExternalScripts: true,
```

Comment:
Here, based on `ExternalScripts` flag docker knows if to look for scripts inside the image or not. Hence I'm reworking this part I'll be clearing this all out.